templates/janus.jcfg

```diff
@@ -199,8 +202,11 @@ media: {
 # be used for gathering candidates, and enable or disable the
 # internal libnice debugging, if needed.
 nat: {
+	{{ if .Env.STUN_SERVER }}
```

Comment:
I'd suggest checking for both STUN_SERVER and STUN_PORT

Reply:
Well, I don't think its necessary - If STUN_SERVER is not set STUN_PORT does nothing and #3 can be fixed by adding a pair of quotes